python/pyspark/sql/session.py

```diff
@@ -550,6 +555,7 @@ def getOrCreate(self) -> "SparkSession":
                 return session
 
         # Spark Connect-specific API
+        @spark_connect_only
```

Comment:
`spark_connect_only` -> `connect_only` or `remote_only` for naming consistency

Reply:
Updated to `remote_only`. Thanks!